shared/naturalcrit/codeEditor/codeEditor.jsx

```diff
@@ -355,12 +355,24 @@ const CodeEditor = createClass({
 				let text = '';
 				let currentLine = from.line;
 				const maxLength = 50;
+
+				let headerText = '';
+				let otherText = '';
```

Comment:
Seems like we don't need both of these since we only use one or the other. Change to `let foldPreviewText = ''` or something?

If we hit a non-header, we store it temporarily, but then keep looking for a header until we find one and we can overwrite this same variable.

Reply:
Applied in the latest iteration